src/consent/consent-debug-geography.enum.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ *  For more information:
+ *  https://developers.google.com/admob/unity/reference/namespace/google-mobile-ads/ump/api#debuggeography
+ *
+ * */
+
+export enum ConsentDebugGeography {
+  /**
+   * Debug geography disabled.
+   */
+  DISABLED = 0,
+
+  /**
+   * Geography appears as in EEA for debug devices.
+   */
+  EEA = 1,
+
+  /**
+   * Geography appears as not in EEA for debug devices.
+   */
+  NOT_EEA = 2,
```

Comment:
those are the values on the native side? Maybe we should use strings here? (easier to debug on web)